src/components/tooltip/NetworkMarker.js

```diff
@@ -69,6 +69,21 @@ function _getHumanReadableDataStatus(status: NetworkStatus): string {
   }
 }
 
+function _getHumanReadableHttpVersion(httpVersion: string): string {
+  switch (httpVersion) {
+    case 'h3':
+      return '3.0';
+    case 'h2':
+      return '2.0';
+    case 'http/1.0':
+      return '1.0';
+    case 'http/1.1':
+      return '1.1';
+    default:
+      return 'Unknown';
```

Comment:
Thanks to the type changes as outlined below, you can replace `return 'Unknown'` with:
```
throw assertExhaustiveCheck(httpVersion, `Unknown received HTTP version ${httpVersion}`);
```

Then later, if a developer adds a new version to the type, Flow will error at this location to make the developer update the switch block as well. And if the developer forgets to add a new version to the types, this will throw here (as an assertion).